prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -373,12 +369,11 @@ class FHIRRouter(
             // TODO: merge with condition filter (see https://github.com/CDCgov/prime-reportstream/issues/12705)
```

Comment:
can be removed?

Reply:
Should probably leave it in until the deployment process finishes and the old code is removed.